CIFAR10/inference/BAIDU-USA-GAIT_ResNet18_1GTX1080_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Jiazhuo Wang, Zhiyu Cheng, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, jiazhuowang@baidu.com, zhiyucheng@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.0.dev20181116",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-ResNet18",

```

Comment:
Please add the commit hash either to the `codeURL` or as a separate `commitHash` field